plugin/src/main/java/net/aufdemrand/denizen/events/player/PlayerMendsItemScriptEvent.java

```diff
@@ -129,15 +129,6 @@ public void onPlayerItemMend(PlayerItemMendEvent event) {
         fire();
         event.setCancelled(cancelled);
         event.setRepairAmount(repairAmount.asInt());
-        final Player p = event.getPlayer();

```

Comment:
did you check to see if you can remove any unneeded imports when you removed this? also did you re-verify the event works properly on 1.12.2 and 1.13.2 after removing this?

Reply:
He didn't (check imports), there are invalid imports indeed! (`Player` is still listed but not needed)

Reply:
I keep forgetting that as much as IntelliJ enjoys adding new imports for statements, it doesn't also remove unused imports.

Give me a moment. I have to rerun the 1.12.2 test (I only confirmed it worked on 1.13.2 before pushing, oops).

Reply:
Tested on 1.12.2 and 1.13.2. Works as expected.